src/styles/blocks/nav.scss

```diff
@@ -0,0 +1,44 @@
+.nav {
+  &__list {
+    display: flex;
+    list-style: none;
+    margin: 0;
+    padding: 0;
+  }
+
+  &__item:last-child a {
+    margin: 0;
+  }
+
+  &__link {
+    display: block;
+    line-height: 60px;
+    font-size: 12px;
+    font-weight: 500;
+    text-transform: uppercase;
+    text-decoration: none;
+    color: $main-text-color;
+    margin-right: 20px;
+    transition: color 0.3s;
+
+    &:hover {
+      color: $blue-brand-color;
+    }
+  }
+
+  .is-active {
+    position: relative;
+    color: $blue-brand-color;
+
+    &::after {
+      content: '';
+      position: absolute;
+      display: block;
+      height: 4px;
+      width: 100%;
+      background-color: $blue-brand-color;
+      border-radius: 8px;
+      bottom: 0;
+    }
+  }
```

Comment:
According to BEM methodology, modifiers should be named using the block or element name followed by two dashes and the modifier name. The class 'is-active' should be named as 'nav__link--is-active' to maintain consistency and clarity in the naming convention. This helps in understanding that 'is-active' is a modifier of the 'nav__link' element.